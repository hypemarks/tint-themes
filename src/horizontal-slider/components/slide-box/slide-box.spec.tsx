import { render } from '@testing-library/react';
import React from 'react';
import SlideBox from './slide-box';

describe('Test <SlideBox /> Component', () => {
  it('should render snapshot', () => {
    const wrapper = render(
      <SlideBox isVerticalOrientation={true}>
        <div>content goes here</div>
      </SlideBox>
    );

    expect(wrapper.asFragment()).toMatchSnapshot();
  });
});
