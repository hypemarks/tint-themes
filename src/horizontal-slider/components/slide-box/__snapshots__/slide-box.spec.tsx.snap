// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <SlideBox /> Component should render snapshot 1`] = `
<DocumentFragment>
  <div
    class="slider__box"
    style="height: 50%;"
  >
    <div>
      content goes here
    </div>
  </div>
</DocumentFragment>
`;
